### PR TITLE
Update urlwatch & Add python3-jq

### PR DESCRIPTION
**Pull request by RunningDroid**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

File: srcpkgs/python3-jq/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'python3-jq'
+pkgname=python3-jq
+version=1.2.2
+revision=1
+wrksrc="${pkgname#*-}.py-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-Cython"
+makedepends="python3-devel jq-devel"
```
Comment (paper42) [inline srcpkgs/python3-jq/template:8]: `$depends` is missing

Comment (RunningDroid) [reply srcpkgs/python3-jq/template:8]: While checking the runtime deps I noticed that the library that python3-jq links against is provided by jq-devel, not jq. Should I change jq's template so those files aren't moved into jq-devel?

Comment (paper42) [reply srcpkgs/python3-jq/template:8]: I meant that python3 is missing from $depends, but that's a good catch. cc @leahneukirchen

**Comment by RunningDroid** [comment]
Some tests in urlwatch-2.25 were changed to assume a more recent version of python3-minidb, so I added that to this PR.

**Pull request merged**